src/hipscat/inspection/visualize_catalog.py

```diff
@@ -58,7 +61,27 @@ def plot_pixels(catalog: Catalog, projection="moll", draw_map=True):
             - orth - Orthographic projection
     """
     pixels = catalog.get_healpix_pixels()
-    max_order = catalog.partition_info.get_highest_order()
+    plot_pixel_list(
+        pixels=pixels,
+        plot_title=f"Catalog pixel density map - {catalog.catalog_name}",
+        projection=projection,
+        draw_map=draw_map,
+    )
+
+
+def plot_pixel_list(pixels: List[HealpixPixel], plot_title: str = "", projection="moll", draw_map=True):
+    """Create a visual map of the pixel density of a list of pixels.
+
+    Args:
+        pixels: healpix pixels (order and pixel number) to visualize
+        plot_title (str): heading for the plot
+        projection (str) The map projection to use. Valid values include:
```

Comment:
Tip of the hat, this renders very nicely in ReadTheDocs 😄

Reply:
Past Melissa fiddled with that for a while. I'm glad it's still paying dividends.